tests/lib/prepare.sh

```diff
@@ -481,9 +481,9 @@ EOF
         # append ubuntu, test user for the testing
         grep "^test:" /etc/$f >> /mnt/system-data/var/lib/extrausers/"$f"
         grep "^ubuntu:" /etc/$f >> /mnt/system-data/var/lib/extrausers/"$f"
-        # check test was copied
-        MATCH "^test:" </mnt/system-data/var/lib/extrausers/"$f"
-        MATCH "^ubuntu:" </mnt/system-data/var/lib/extrausers/"$f"
+        # check users were copied
+        grep "^test:" /mnt/system-data/var/lib/extrausers/"$f"
```

Comment:
I think there's something different at play here. With MATCH this expanded to  something in the lines of:
```
grep -q -E "^test:" </mnt/system-data/var/lib/extrausers/"$f"
```